use_case_examples/hybrid_model/compile_hybrid_llm.py

```diff
@@ -0,0 +1,140 @@
+"""Showcase for the hybrid model converter."""
+
+import os
+from copy import deepcopy
+from pathlib import Path
+from typing import List, Union
+
+import torch
+from concrete.fhe import Configuration, ParameterSelectionStrategy
+from datasets import load_dataset
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from concrete.ml.torch.hybrid_model import HybridFHEModel
+
+
+def compile_model(
+    model_name: str,
+    model: torch.nn.Module,
+    inputs: torch.Tensor,
+    module_names: Union[str, List],
+    expected_accuracy,
+    models_dir: Path,
+):
+    """Run the test for any model with its private module names."""
+
+    # Enable multi params/precision
+    configuration = Configuration(
+        single_precision=False,
+        parameter_selection_strategy=ParameterSelectionStrategy.MULTI,
+    )
+
+    # Create a hybrid model
+    hybrid_model = HybridFHEModel(model, module_names)
+    hybrid_model.compile_model(
+        inputs,
+        n_bits=8,
+        # setting it to None is not enough -> weird
```

Comment:
Can we remove? What did you mean here?

Reply:
Hum not sure on that, I had a weird behavior if not set, I'll try removing it

Reply:
Are you talking about rounding_threshold_bits? Isn't it default to None anyway?

Reply:
Removed the comment, it was outdated